Makefile

```diff
@@ -101,7 +101,7 @@ podman: .gopathok $(PODMAN_VARLINK_DEPENDENCIES)
 	$(GO) build -i -ldflags '$(LDFLAGS_PODMAN)' -tags "$(BUILDTAGS)" -o bin/$@ $(PROJECT)/cmd/podman
 
 darwin:
-	GOOS=darwin $(GO) build -ldflags '$(LDFLAGS_PODMAN)' -tags "$(BUILDTAGS)" -o bin/darwin$@ $(PROJECT)/cmd/podman
+	GOOS=darwin $(GO) build -ldflags '$(LDFLAGS_PODMAN)' -tags "containers_image_ostree_stub containers_image_openpgp" -o bin/podman.$@ $(PROJECT)/cmd/podman
```

Comment:
We should have a variable to hold the tags - BUILDTAGS_OSX or similar - so people can set them via environment variable when building

Reply:
agree

Reply:
> We should have a variable to hold the tags - BUILDTAGS_OSX...

Or pick conservative tags, like I've dond in #1034.

Reply:
@wking i assumming your PR will be rebased on mine and clean that up